### ISSUE TITLE
Exception on getMyCommands method

### ISSUE THREAD

**Issue by hamidqp** [badfarm/zanzara#44]
code: 
```php
$ctx->getMyCommands()->then(fn($result) => var_dump($result), fn($result) => var_dump($result));
```

result: 
```

  ["message":protected]=>
  string(29) "Class "Scalar" does not exist"
  ["string":"Exception":private]=>
  string(0) ""
  ["code":protected]=>
  int(-1)
  ["file":protected]=>
  string(78) "/vendor/netresearch/jsonmapper/src/JsonMapper.php"
  ["line":protected]=>
  int(655)
  ["trace":"Exception":private]=>
  array(45) {
    [0]=>
    array(5) {
      ["file"]=>
      string(78) "/vendor/netresearch/jsonmapper/src/JsonMapper.php"
      ["line"]=>
      int(655)
      ["function"]=>
      string(11) "__construct"
      ["class"]=>
      string(15) "ReflectionClass"
      ["type"]=>
      string(2) "->"
    }
    [1]=>
    array(5) {
      ["file"]=>
      string(78) "/vendor/netresearch/jsonmapper/src/JsonMapper.php"
      ["line"]=>
      int(449)
      ["function"]=>
      string(14) "createInstance"
      ["class"]=>
      string(10) "JsonMapper"
      ["type"]=>
      string(2) "->"
    }
    [2]=>
    array(5) {
      ["file"]=>
      string(82) "/vendor/badfarm/zanzara/src/Zanzara/ZanzaraMapper.php"
      ["line"]=>
      int(54)
      ["function"]=>
      string(8) "mapArray"
      ["class"]=>
      string(10) "JsonMapper"
      ["type"]=>
      string(2) "->"
    }
    [3]=>
    array(5) {
      ["file"]=>
      string(91) "/vendor/badfarm/zanzara/src/Zanzara/Telegram/TelegramTrait.php"
      ["line"]=>
      int(1873)
      ["function"]=>
      string(9) "mapObject"
      ["class"]=>
      string(21) "Zanzara\ZanzaraMapper"
      ["type"]=>
      string(2) "->"
    }
    [4]=>
    array(5) {
      ["file"]=>
      string(75) "/vendor/react/promise/src/FulfilledPromise.php"
      ["line"]=>
      int(28)
      ["function"]=>
      string(26) "Zanzara\Telegram\{closure}"
      ["class"]=>
      string(15) "Zanzara\Context"
      ["type"]=>
      string(2) "->"
    }
    [5]=>
    array(5) {
      ["file"]=>
      string(66) "/vendor/react/promise/src/Promise.php"
      ["line"]=>
      int(134)
      ["function"]=>
      string(4) "then"
      ["class"]=>
      string(30) "React\Promise\FulfilledPromise"
      ["type"]=>
      string(2) "->"
    }
...
```

**Comment by cheeghi** [comment]
Hi, thanks a lot for the report! I fixed it on develop branch. I will release it soon.

**Comment by cheeghi** [comment]
I released a **0.7.1** with the fix. Let me know if it works, thank you!
